packages/jest-core/src/runJest.ts

```diff
@@ -73,15 +74,55 @@ type ProcessResultOptions = Pick<
   Config.GlobalConfig,
   'json' | 'outputFile' | 'testResultsProcessor'
 > & {
-  collectHandles?: () => Array<Error>;
+  collectHandles?: (keepOpen: boolean) => Array<Error>;
   onComplete?: (result: AggregatedResult) => void;
   outputStream: NodeJS.WriteStream;
 };
 
-const processResults = (
+const wait = promisify(setTimeout);
```

Comment:
can we move this into `createOpenHandlesResult` so it doesn't affect all test runs?

Reply:
affect how? this just creates a new function in the module scope

Reply:
ah, CI complains. Doesn't locally though...

Reply:
This is likely not an issue at al. The point is, if we can avoid calling something, for a regular user in this scenario, why not doing so. Even if it's a small thing, they like to add up.